pkgs/top-level/all-packages.nix

```diff
@@ -30134,7 +30134,11 @@ with pkgs;
 
   legitify = callPackage ../development/tools/legitify { };
 
-  lens = callPackage ../applications/networking/cluster/lens { };
+  lens =
+    if stdenv.isDarwin then
+      callPackage ../applications/networking/cluster/lens/darwin.nix { }
+    else
+      callPackage ../applications/networking/cluster/lens { };
```

Comment:
I would prefer moving this logic to `default.nix`, while renaming the original `default.nix` to `linux.nix`.